packages/forms-web-app/src/pages/projects/register/agent/about-project/controller.js

```diff
@@ -66,7 +63,7 @@ const postRegisterAgentAboutProjectController = async (req, res) => {
 				mode: req.session.mode
 			});
 
-			if (commentsData) await postCommentsData(ipRefNo, commentsData);
+			if (commentsData) await putComments(ipRefNo, commentsData);
```

Comment:
Any reason to change this to a PUT?  I am wondering if there could be side affects to this, not sure if PUT returns a reference ID, I think the PUT is deprecated in the API as well.